pages.ko/common/asciinema.md

```diff
@@ -0,0 +1,36 @@
+# asciinema
+
+>  터미널 세션을 녹음하고 재생하며 선택적으로 asciinema.org에서 공유합니다. 
+> 더 많은 정보: <https://asciinema.org/>.
+
+- `asciinema`  로컬 설치와 with an asciinema.org 계정을 연결하기:
+
+`asciinema auth`
+
+- 새로운 녹음을 작성 (한 번 완료되면 사용자는 업로드하거나 로컬로 저장하라는 메시지가 표시됩니다):
+
+`asciinema rec`
+
+- 새 녹음을 만들고 로컬 파일에 저장:
+
+`asciinema rec {{path/to/file}}.cast`
```

Comment:
Perhaps `path/to/file` here could be translated too? `cast_id`, `name`, `version` etc. in the `{{token syntax}}` too for example below. Basically anything in the `{{token_syntax}}` that doesn't affect the execution of the command can be translated if possible.

Reply:
Of course you can translate path/to/file. However, in Korea, most folder or file names are in English. But I'll translate it if you want.

Reply:
Ah, I see. In other languages, we prefer to translate them. However, if in Korea you usually have file and folder names in English, I guess we should stick with the cultural norms.

Not speaking Korean means that I can only suggest things that might help :P

Reply:
Korean programmers prefer to use English as it is for commands or terminology in computers (eg file names). But if there is no problem, I will translate it. Thank you for your comment!